### PR TITLE
Fix using bundle configured `nbResults` for search

### LINKED ISSUE #306: `nbResults` is not set / ignored
**Description**
`nbResults` is ignored

**Expected behavior**
Set default max number of results per search

**Current behavior**
Defaults to always `20`

**Screenshots or Logs**
`meilisearch.yaml` : 

```yaml
meilisearch:
  url: http://meili:7700
  nbResults: 5
```

the issue can be resolved by setting manually a `limit` like

```php
            $result = $this->meiliSearchService->search(
                objectManager: $this->entityManager,
                className: PriceList::class,
                query: $searchTerm,
                searchParams: [
                    'limit' => 5,
                    'filter' => sprintf('tenant.name = "%s"', $tenant->getName())
                ]
            );
```

### PR DESCRIPTION
**Pull request by norkunas**
# Pull Request

## Related issue
Fixes #306

## What does this PR do?
- Provides configured `nbResults` to engine search when it's not explicitly provided manually

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

File: src/Services/MeilisearchService.php [hunk 1]
```diff
@@ -155,7 +155,7 @@ public function search(
     ): array {
         $this->assertIsSearchable($className);
 
-        $ids = $this->engine->search($query, $this->searchableAs($className), $searchParams);
+        $ids = $this->engine->search($query, $this->searchableAs($className), $searchParams + ['limit' => $this->configuration['nbResults']]);
```
Comment (brunoocasali) [inline src/Services/MeilisearchService.php:158]: Shouldn't we use an `array_merge` or something like that?

Comment (norkunas) [reply src/Services/MeilisearchService.php:158]: No need :) test proves that:)

**Review by brunoocasali — APPROVED** [review]
LGTM then!!!

**Comment by brunoocasali** [comment]
bors merge

**Pull request merged**